Gemfile

```diff
@@ -42,6 +42,7 @@ group :development, :test do
   gem 'launchy'
   gem 'shoulda-matchers', '~> 3.0'
   gem 'fabrication'
+  gem 'faker', github: 'stympy/faker.git'
```

Comment:
@meyerhoferc @CjMoore I made and committed this change, should ensure that the faker gem works correctly.